electron/custom_install_steps.nsh

```diff
@@ -26,32 +43,63 @@
     nsExec::ExecToStack 'add_tap_device.bat i386'
   ${EndIf}
 
-  
   Pop $0
   Pop $1
   StrCmp $0 0 installservice
-  MessageBox MB_OK "Sorry, we could not configure your system to connect to Outline. Please try running the installer again.$\n$\nIf you still cannot install Outline, please get in touch with us and let us know that the TAP device failed to install with this error: $1"
-  ; TODO: Abort gracefully, i.e. uninstall, before exiting.
-  Quit
+  MessageBox MB_OK "Sorry, we could not configure your system to connect to Outline. Please try \
+    running the installer again. If you still cannot install Outline, please get in \
+    touch with us and let us know that the TAP device failed to install with this error:$\n$\n$1"
 
-  installservice:
+  ; Submit a Sentry error event.

```

Comment:
Should we refactor this logic so that we also send a report when on service installation failures?

Reply:
Absolutely. I didn't do it this time just because we're already getting that information whe the client restarts the service. Just added a comment to remove (or at least revisit) that, and have the installer trigger an event.